recipes/earthengine-api/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "earthengine-api" %}
+{% set version = "0.1.137" %}
+{% set sha256 = "6dd2e0e8de8214c5cf499c55f13154f6a5109ef5ea673260c80b22f9a2b51c96" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - earthengine = ee.cli.eecli:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - google-api-python-client
+    - pyopenssl >=0.11
+  run:
+    - python
+    - google-api-python-client
+    - pyopenssl >=0.11
+
+test:
+  imports:
+    - ee
+    - ee.cli
+  commands:
+    - earthengine --help
+
+about:
+  home: https://earthengine.google.com/
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
To use the `license_file` that you've included, you typically need to use the syntax specified in the link below so that conda knows where to look for the file:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

It's not causing an error, which is strange because it's not included in the source distribution. I noted this behavior in another recent pull request, so I'm not sure what's going on:

https://github.com/conda-forge/staged-recipes/pull/5740#discussion_r184687747

Reply:
Ok, this is actually fine. There's a new feature of conda-build that I was previously unaware of https://github.com/conda-forge/staged-recipes/issues/5749

Reply:
Acknowledged.